Backpack-SwiftUI/Dialog/Classes/DialogTextContent.swift

```diff
@@ -44,6 +44,8 @@ struct DialogTextContent: View {
                     Spacer()
                 }
                 BPKText(title, style: .heading3)
+                    .lineLimit(nil)
+                    .multilineTextAlignment(.center)
```

Comment:
shouldnt this modifier be applied only if the contentAlignment is .center?

Reply:
or maybe, using the contentAlighnment value?

Reply:
I don't think so, it's a title view, it would be really weird looking if it's aligned to right or left (I tried😅). Also, I checked how it's implemented in [UIKit](https://github.com/Skyscanner/backpack-ios/blob/f6347b8801627525fec6a9e6c811a73883dfe61d/Backpack/Dialog/Classes/BPKDialogContentView.m#L91) and [Android Backpack](https://github.com/Skyscanner/backpack-android/blob/931d605b250975db87b13dc72fbf27f0388f433d/Backpack/src/main/res/layout/bpk_dialog.xml#L43), both set it to center alignment always.